src/Raven.Server/Rachis/Elector.cs

```diff
@@ -54,6 +52,17 @@ public void HandleVoteRequest()
                             clusterTopology = _engine.GetTopology(context);
                         }
 
+                        if (clusterTopology.TopologyId == null)
+                        {
+                            _connection.Send(context, new RequestVoteResponse
+                            {
+                                Term = rv.Term,
+                                VoteGranted = true,
+                                Message = "I might vote for you, because I'm not part of any cluster."
```

Comment:
This not a tryout, this is an _actual_ vote. The text needs to reflect this

Reply:
done